src/theory/bir/bir_exp_memScript.sml

```diff
@@ -613,7 +613,12 @@ val bir_update_mmap_def = Define `
     (!mmap aty a.      (bir_update_mmap aty mmap a [] = mmap)) /\
     (!mmap aty a v vs. (bir_update_mmap aty mmap a (v::vs) =
                         bir_update_mmap aty (FUPDATE mmap ((bir_mem_addr aty a), v2n v)) (SUC a) vs))`;
-
+(* TODO: Order of updates in memory representation
+val bir_update_mmap_rev_def = Define `
```

Comment:
I'm curious, why is the rev update necessary?

Reply:
It's not necessary, it's older WIP material which was accidentally left in a comment :stuck_out_tongue:

I've removed it now, I'll push a new commit soon to address your comments.